test/unit/helpers/xhr/PASSWORD_RESET_withComplexity.js

```diff
@@ -7,6 +7,10 @@ define({
     "status": "PASSWORD_RESET",
     "_embedded": {
       "policy": {
+        "age": {
+          "minAgeMinutes": -1,
+          "historyCount": 0
```

Comment:
should we set this to ```7``` here and that way we don't have to add that value in the test itself? Also that will help in case anyone else uses this xhr in some other test